sdk/servicebus/service-bus/README.md

```diff
@@ -134,11 +134,11 @@ You can use this receiver in one of 3 ways to receive messages:
 
 #### Get an array of messages
 
-Use the [receiveBatch][receiverreceivebatch] function which returns a promise that
+Use the [receiveMessages][receiverreceivebatch] function which returns a promise that
```

Comment:
[receiverreceivebatch] will need to change and I think that'll also require a change to the URLs to the docs as well.

Reply:
I left this as is because it would point to a broken link. Since we do a clean up of links when we get closer to release, we could do it then.